README.md

```diff
@@ -235,15 +235,48 @@ expect(any_callable).to.throw(Exception)
 Asserts that a mock has been called
 
 ```python
-expect(mock).to.be.called
+expect(mock).to.be.called()
+```
+
+#### called_once
+
+Asserts that a mock has been called exactly one time
+
+```python
+expect(mock).to.be.called_once()
 ```
 
 #### callable
 
 Asserts that a object is callable 
 
 ```python
-expect(object).to.be.callable
+expect(object).to.be.callable()
+```
+
+#### called_with
+
+Asserts that a mock has been called with params 
+
+```python
+expect(mock).to.be.called_with(*args, **kwargs)
+```
+
+#### called_once_with
+
+Asserts that a mock has been called once with params 
+
+```python
+expect(mock).to.be.called_once_with(*args, **kwargs)
+```
+
+#### ever_called_with
+
+Asserts that a mock has ever been called with params. 
+The call is not necessary to be to latest one (the same as assert.any_call).
+
+```python
+expect(mock).to.have.ever.been.ever_called_with(*args, **kwargs)
```

Comment:
This writing is somewhat redundant. Could we rewrite it with "have been ever called with"? Or, try to support the case of `have.ever.been.called_with` alone?